<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PopupDialCorpoFact</class>
 <widget class="QWidget" name="C_PopupDialCorpoFact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>118</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_kg_x">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>0</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> mg x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>326</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_unit_FactCorpo">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>0</y>
     <width>136</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FactCorpo">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>0</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>mode fractionnaire simple (sur l'unité de prise)</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QPA_by_UP">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>95</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UPA">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>35</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>mg</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QPA">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UFC">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>55</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kg</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>40</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité de principe actif</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_QFC">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_QFC">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>à administrer par (facteur corporel)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA_by_UP_0">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>85</y>
     <width>166</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité de principe actif </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_QPA_by_UP_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>156</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>par unité de prise</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_secable">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>95</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>arrondi au quart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
